testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/db/DbClientMetricsTestUtil.java

```diff
@@ -19,6 +19,9 @@ public static void assertDurationMetric(
       InstrumentationExtension testing,
       String instrumentationName,
       AttributeKey<?>... expectedKeys) {
+    // db.system is required - see
+    // https://opentelemetry.io/docs/specs/semconv/database/database-metrics/#metric-dbclientoperationduration
+    assertThat(expectedKeys).extracting(AttributeKey::getKey).contains("db.system");
```

Comment:
is there a reason not to use the constant here?

```suggestion
    assertThat(expectedKeys).extracting(AttributeKey::getKey).contains(DB_SYSTEM);
```